openlibrary/plugins/inside/code.py

```diff
@@ -65,9 +70,7 @@ def inside_search_select(params):
                 return {'error': 'Error converting search engine data to JSON'}
 
         def quote_snippet(snippet):
-            trans = { '\n': ' ', '{{{': '<b>', '}}}': '</b>', }
-            re_trans = re.compile(r'(\n|\{\{\{|\}\}\})')
-            return re_trans.sub(lambda m: trans[m.group(1)], web.htmlquote(snippet))
+            return self.quote_snippet(snippet)
```

Comment:
I pulled out `quote_snippet` to make it testable, but wanted to leave the rest of the structure as unchanged as possible in case anything expected it at the previous location. You are right though that it only used once, so may as well be directly passed. 

`return render_template('search/inside.tmpl', get_results, self.quote_snippet, editions_from_ia, read_from_archive)`

I'm happy to change it to reduce unnecessary code.